uavcan/file/405.GetInfo.1.0.uavcan

```diff
@@ -0,0 +1,23 @@
+#
```

Comment:
This is changed from the old DSDL. It should be in v0.1 initially. We can, however, make a priority to test and stabilize this definition.

The file stuff has high priority, but we should prioritize the protocol stuff more. Let's make everything in file v0.1 initially and then do some testing and stabilize relatively quick. 

I don't want the essential stuff in the node namespace to have to wait for the file definitions before we are able to merge the PR.

Reply:
Will change the version to 0.